ietf/doc/models.py

```diff
@@ -523,9 +524,81 @@ def replaces_canonical_name(self):
     def replaced_by(self):
         return set([ r.document for r in self.related_that("replaces") ])
 
+    def render_txt_and_html(self):
```

Comment:
This is going to take too long to run - users will get timeouts from the cdn,similar to what they get now sometimes posting a draft via the UI instead of the async API. What needs to happen here instead is to dispatch work to the celery containers we have now instead, doing something reasonable for _this_ attempt to view (Probably best to show whatever we _have_ rather than return a "Check back in a few minutes" or the like). But we have to build that scaffolding first.